mokapot/model.py

```diff
@@ -290,8 +315,8 @@ def fit(self, psms):
             norm_feat = norm_feat[shuffled_idx, :]
             start_labels = start_labels[shuffled_idx]
```

Comment:
Move this to above the added code.

Reply:
I would need to move the entire code block under "Shuffle order" correct?